litellm/caching.py

```diff
@@ -64,10 +64,16 @@ async def disconnect(self):
 
 
 class InMemoryCache(BaseCache):
```

Comment:
Does the cache in Redis keep growing and need regular cleanup?

Reply:
yes it does - perhaps we can set a default ttl on redis caching. Planning on addressing that in a separate PR